### DISCUSSION TITLE
Blog post on organization public page

### DISCUSSION THREAD

**Discussion by abmmhasan** [gautamkrishnar/blog-post-workflow#253] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Yes

**Describe the solution you'd like**
How to implement this in organization public page of Github?

**Describe alternatives you've considered**
Still not one!

**Additional context**
N/A

### Double check the following
Before creating a feature request, Please make sure that you had done the following, add a checkmark if done by marking it with `[x]`:

- [ x] This feature cannot be accomplished using any of the available [options](https://github.com/gautamkrishnar/blog-post-workflow#options) provided by the workflow.
- [ x] I had followed all the steps mentioned in the [documentation](https://github.com/gautamkrishnar/blog-post-workflow)

**Comment by gautamkrishnar** [comment]
Just check the example of cisco devnet https://github.com/CiscoDevNet

**Reply by abmmhasan** [reply, marked as answer]
Well, I had to provide readme_path for it
```
readme_path: "./profile/README.md"
```
